app/views/searches/search.html.erb

```diff
@@ -0,0 +1,72 @@
+<div class="row">
+  <% if !@provider.nil? %>
+  <div class="jumbotron">
+    <h1>
+      <%= @provider.capitalize %> Search Results
+    </h1>
+  </div>
+  <% else%>
+    <div class="jumbotron">
+      <h1>Please select Twitter or Vimeo</h1>
+    </div>
+<!-- </div> -->
+<% end %>
+<% if !@results.nil? %>
+  <% @results.in_groups_of(3, false) do |group| %>
+    <div class="row">
+      <% group.each do |result| %>
```

Comment:
Since bootstrap includes ways of managing rows and columns, I'd suggest using bootstrap to split your display into rows of 3 instead of `in_groups_of`. Bootstrap is really good at this! :tada: